notebooks/README.md

```diff
@@ -3,7 +3,11 @@ This directory contains *Jupyter notebooks* illustrating how to use Certifai in
 
 ### Azure ML Model Headers Demo
 
- Azure Model headers demo example notebook highlights end-to-end process of running a Certifai Fairness evaluation by starting out with training, testing and deploying a sklearn model as a web service with token based Authentication on Azure, using Azure Containerized Instance(ACI) and launching a Certifai model scan on the deployed model in the same notebook. The example notebook can be run either on local machine or on Azure hosted notebook environments.
+Azure Model headers demo example notebook highlights end-to-end process of running a Certifai Fairness evaluation by starting out with training, testing and deploying a sklearn model as a web service with token based Authentication on Azure, using Azure Containerized Instance(ACI) and launching a Certifai model scan on the deployed model in the same notebook. The example notebook can be run either on local machine or on Azure hosted notebook environments.
```

Comment:
Change to: "The Azure Model headers demo example notebook highlights the end-to-end process of running a Certifai fairness evaluation by starting with training, testing, and deploying a sklearn model as a web service with token-based Authentication on Azure, using Azure Containerized Instance(ACI) and launching a Certifai model scan on the deployed model in the same notebook. The example notebook can be run either on local machine or on an Azure-hosted notebook environments."

Reply:
Done.